agent/security_helper.py

```diff
@@ -53,7 +61,7 @@ def netstat_scan():
         'remote_address': c.raddr,
         'status': c.status if c.type == socket.SOCK_STREAM else None,
         'pid': c.pid
-    } for c in connections]
+    } for c in connections if c.raddr and c.laddr]
```

Comment:
Probably we shouldn't skip connections with empty local address info. They still can be useful for blocking remote connections

Reply:
@rptrchv if API can handle it now, I'll revert this

Reply:
yes, it can